### ISSUE TITLE
Temporary 404s with renamed databases

### ISSUE THREAD

**Issue by msakrejda** [heroku/heroku-pg#102]
Because we contact the Shogun API using add-on names rather than their uuids, for a period of time after a rename, Shogun is unable to find the add-on. Due to related sync issues in Shogun, we have some [heuristic error handling](https://github.com/heroku/heroku-pg/blob/43d39f967138760a3ec2fdbf5f8821ecd9e4fe56/commands/info.js#L63-L64) where anything that 404s is "not yet provisioned". This means we give a misleading error message:

```
➜ h addons:rename --app sushi postgresql-shiny-12345 sushi-primary
postgresql-shiny-12345 successfully renamed to sushi-primary.
➜ h pg:info --app sushi
 ▸    sushi-primary is not yet provisioned.
 ▸    Run heroku pg:wait to wait until the db is provisioned.
```

Changing the client to using uuids to contact Shogun would work around this problem.

/cc @mikelikesbikes

**Comment by msakrejda** [comment]
This should be fixed by #103

**Comment by msakrejda** [comment]
#103 was merged and will be released with the next CLI release.
